admin/tools/docker/deployment/in2p3/examples/chown.sh

```diff
@@ -0,0 +1,3 @@
+set -x
+
+/opt/shmux/bin/shmux -c "echo \"$PASSWORD\" | sudo -S sh -c 'chown -R qserv /qserv/data'" ccqserv{125..149}.in2p3.fr
```

Comment:
It's only an example in case someone need to write scripts which require shmux + root access. Comment added.